### PR TITLE
Adding in message_type support for FTP logging endpoint

### PR DESCRIPTION
**Pull request by annez**
Relates to: https://github.com/fastly/go-fastly/pull/212

### CODE REVIEW

File: fastly/block_fastly_service_v1_logging_ftp_test.go [hunk 1]
```diff
@@ -265,12 +269,13 @@ resource "fastly_service_compute" "foo" {
...
     user       = "user"
-	public_key = file("test_fixtures/fastly_test_publickey")
+	  public_key = file("test_fixtures/fastly_test_publickey")
     password         = "p@ssw0rd"
     path             = "/path"
     port             = 27
     gzip_level       = 3
-    timestamp_format = "%%Y-%%m-%%dT%%H:%%M:%%S.000"
+		timestamp_format = "%%Y-%%m-%%dT%%H:%%M:%%S.000"
+		message_type     = "classic"
```
Comment (phamann) [inline fastly/block_fastly_service_v1_logging_ftp_test.go]: Looks like there are some whitespace issues here.

**Review by phamann — APPROVED** [review]
LGTM - other than the minor whitespace nit.

File: fastly/block_fastly_service_v1_logging_ftp_test.go (hunk 1 above)
Comment (annez) [reply fastly/block_fastly_service_v1_logging_ftp_test.go]: It switched them to tabs which is why it got confused. Fixed

**Review by phamann — APPROVED** [review]
LGTM

**Pull request merged**